core/arch/arm/plat-mediatek/main.c

```diff
@@ -33,7 +35,9 @@ void boot_primary_init_intc(void)
 
 void console_init(void)
 {
+#if defined(CFG_CONSOLE_ENABLE)
```

Comment:
```
if (CFG_TEE_CORE_LOG_LEVEL != 0) {
    ...
}
```

Reply:
what about using `#if (CFG_TEE_CORE_LOG_LEVEL != 0)` here ?

I guess we make the assumption that the compiler will detect/remove the code around bracket ?

Reply:
> what about using `#if (CFG_TEE_CORE_LOG_LEVEL != 0)` here ?
> 
> I guess we make the assumption that the compiler will detect/remove the code around bracket ?

Exactly. It is a general rule to to improve compile test coverage, see 776d21586. The Linux kernel has a similar policy.